src/lib/Persistence/Legacy/Content/Gateway/DoctrineDatabase.php

```diff
@@ -1609,6 +1609,20 @@ public function insertRelation(RelationCreateStruct $createStruct): int
         return (int)$this->connection->lastInsertId(self::CONTENT_RELATION_SEQ);
     }
 
+    public function loadRelation(int $relationId): array
```

Comment:
Missing Doctrine exceptions:
```suggestion
    /**
     * @throws \Doctrine\DBAL\Driver\Exception
     * @throws \Doctrine\DBAL\Exception
     */
    public function loadRelation(int $relationId): array
```